src/lib/action.ts

```diff
@@ -237,10 +238,10 @@ More details printed to console.`
       return "invalid latex";
     }
 
-    const img = await this.canvas.addImage(
+    const img: FabricTeXImage = await this.canvas.addImage(
```

Comment:
This isn't really necessary (we don't care about how much stronger this type is than `Image`) but it conveys intent